Classes/Hooks/PageRenderer/TrackingCodeInjector.php

```diff
@@ -70,7 +74,8 @@ public function execute(?array &$params, PageRenderer $pageRenderer): void
         if ($configuration->tagManagerContainerId !== '') {
             $scriptCode .= $this->tagManagerCodeBuilder->setConfiguration($configuration)->getCode();
         }
-        $pageRenderer->addHeaderData("<script>{$scriptCode}</script>");
+
+        $pageRenderer->addHeaderData($this->scriptTagBuilder->build("{$scriptCode}"));
```

Comment:
Looks funny: `"{$scriptCode}"`. Think `$scriptCode` is enough ;-)

Reply:
Okay, that would be a change within the code (see red line)

Reply:
Jup, see left side: It was used in a string. Now you can just pass the variable on ...